src/amo/components/CollectionAddAddon/styles.scss

```diff
@@ -1,8 +1,24 @@
 @import '~photon-colors/photon-colors';
 @import '~amo/css/inc/vars';
+@import '~core/css/inc/mixins';
 
 .CollectionAddAddon {
-  margin-bottom: 24px;
+  margin: $padding-page 0;
+
+  @include respond-to(large) {
+    margin-bottom: $padding-page-l;
+    margin-top: 0;
+  }
+
+  .Card-contents {
+    overflow: visible;
+    // 6px is the margin from the CollectionAddAddon-noticePlaceholder.
+    padding-top: calc(20px - 6px);
```

Comment:
do we need calc here? can this just be `padding-top: 14px`

Reply:
I think it's kinda nice to see see that 20px is the starting point. It might be helpful when we finally fix all 20px margins 😛 They don't follow the pixel spec.

Reply:
Yeah, I did that to be explicit about how the 14px is calculated.